examples/webgl_materials_instance_uniform_nodes.html

```diff
@@ -0,0 +1,215 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - material instance uniform</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+		<link type="text/css" rel="stylesheet" href="main.css">
+	</head>
+	<body>
+		<div id="info">
+			<a href="https://threejs.org" target="_blank" rel="noopener">three.js</a> - webgl material instance uniform
+		</div>
+
+		<script type="importmap">
+		{
+			"imports": {
+				"three": "../build/three.module.js"
+			}
+		}
+		</script>
+		<script type="module">
+
+			import * as THREE from 'three';
+
+			import Stats from './jsm/libs/stats.module.js';
+
+			import { nodeFrame } from './jsm/renderers/webgl/nodes/WebGLNodes.js';
+			import { NodeUpdateType } from './jsm/renderers/nodes/core/constants.js';
+			import Node from './jsm/renderers/nodes/core/Node.js';
+			import ColorNode from './jsm/renderers/nodes/inputs/ColorNode.js';
+
+			class InstanceUniformNode extends Node {
+
+				constructor() {
+
+					super( 'vec3' );
+
+					this.updateType = NodeUpdateType.Object;
+
+					this.inputNode = new ColorNode();
+
+				}
+
+				update( frame ) {
+
+					const rendererState = frame.renderer.state;
+					const mesh = frame.object;
+
+					const meshColor = mesh.color;
+
+					this.inputNode.value.copy( meshColor );
+
+					// force refresh material uniforms
+					rendererState.useProgram( null );
+
+				}
+
+				generate( builder, output ) {
+
+					return this.inputNode.build( builder, output );
+
+				}
+
+			}
+
+			let stats;
+
+			let camera, scene, renderer;
+			let pointLight;
+
+			const objects = [];
+
+			init();
+			animate();
+
+			function init() {
+
+				const container = document.createElement( 'div' );
+				document.body.appendChild( container );
+
+				camera = new THREE.PerspectiveCamera( 45, window.innerWidth / window.innerHeight, 1, 2000 );
+				camera.position.set( 0, 200, 800 );
+
+				scene = new THREE.Scene();
+
+				// Grid
+
+				const helper = new THREE.GridHelper( 1000, 40, 0x303030, 0x303030 );
+				helper.position.y = - 75;
+				scene.add( helper );
+
+				// Material
+
+				const instanceUniform = new InstanceUniformNode();
+
+				const material = new THREE.MeshStandardMaterial();
+				material.colorNode = instanceUniform;
```

Comment:
Hmm, `THREE.MeshStandardMaterial` doesn't have this property... 🤔

Reply:
In new `NodeMaterial` system all `albedo(.map)` and `.color` properties is used as `Material.colorNode` like:
https://github.com/mrdoob/three.js/blob/f07a31a0ed2dca28579816bf75aeb3da6f42950b/examples/webgl_materials_standard_nodes.html#L99-L105

Reply:
Shouldn't it be `const material = new MeshStandardNodeMaterial();` then?

Reply:
I think that yes, It would be better for serialization to extend. Currenty this pattern works for any material that use `color` and `map` like: `MeshBasicMaterial`, `MeshPhongMaterial` `MeshPhysicalMaterial`... would we then have to extended all the materials? The new `NodeMaterial` system not need more special materials like `StandardNodeMaterial`.

Reply:
I can do this in an other PR.

Reply:
>  The new NodeMaterial system not need more special materials like StandardNodeMaterial.

Yeah, that was the original design idea. Meaning just  monkey-patch the existing material classes with new node properties like `colorNode`. The default material properties like `color` are going to internally mapped to nodes for backwards compatibility reasons. Meaning an existing material definition from WebGL also works with WebGPU. However user can start with node features by just adding node properties to the material.

I guess this a good opportunity to clarify whether we want to use this approach or introduce separate node material classes again (like with the previous node system).

Reply:
> Yeah, that was the original design idea. Meaning just  monkey-patch the existing material classes with new node properties like `colorNode`. The default material properties like `color` are going to internally mapped to nodes for backwards compatibility reasons.

Hmm, I think that's a bit confusing at this point... I would keep the materials separate for now.